plugins/woocommerce/src/Internal/Admin/RemoteFreeExtensions/GetRuleProcessor.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Automattic\WooCommerce\Internal\Admin\RemoteFreeExtensions;
+
+use Automattic\WooCommerce\Admin\RemoteSpecs\RuleProcessors\GetRuleProcessor as OriginalGetRuleProcessor;
+use Automattic\WooCommerce\Admin\RemoteSpecs\RuleProcessors\RuleProcessorInterface;
+use Automattic\WooCommerce\Internal\Admin\RemoteFreeExtensions\Rules\ContextPluginsRuleProcessor;
+use Automattic\WooCommerce\Internal\Admin\RemoteFreeExtensions\Rules\ContextVarsRuleProcessor;
+
+/**
+ * A custom GetRuleProcessor class to support context_vars and context_plugins rule types.
```

Comment:
Suggestion: `context_vars` could be completely removed if there is no current use-case for it (assuming there will be a separate prop for mobile ordering).

Reply:
@adrianduffell Yup! Sounds good 👍